### PR TITLE
Update DNS hosting guidance

### PR DESCRIPTION
**Pull request by soupdragon99**
Updated guidance to match GOV.UK guidelines for style and readability.

### CODE REVIEW

**Comment by soupdragon99** [comment]
Updated guidance to GOV.UK style, could do with a pre 2i sanity check @tomnatt thanks

File: source/standards/dns-hosting.html.md.erb [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 
+As part of your DNS strategy and planning we recommend your:
 
-## Principles
+* DNS entry configuration comes from a version-controlled repository such as GitHub, unless autogenerated
+* overall service strategy includes nameserver availability
 
-- DNS entry configuration should come from a version-controlled repository unless autogenerated
-- Consider nameserver availability as part of your overall service availability
+When you implement your DNS strategy we recommend using:
```
Comment (tomnatt) [inline source/standards/dns-hosting.html.md.erb]: This can be stronger I think: "you should use"

File: source/standards/dns-hosting.html.md.erb [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
 
+* [Amazon Route 53](https://aws.amazon.com/route53) as a cloud DNS web service
+* [Google Cloud DNS](https://cloud.google.com/dns/) or [Dyn](https://dyn.com) if you want greater nameserver availability
```
Comment (tomnatt) [inline source/standards/dns-hosting.html.md.erb]: Are we still using Dyn? Can you check these options with @stephenharker?

File: source/standards/dns-hosting.html.md.erb (hunk 1 above)
Comment (soupdragon99) [reply source/standards/dns-hosting.html.md.erb]: Agreed, have now changed this.

File: source/standards/dns-hosting.html.md.erb (hunk 2 above)
Comment (stephenharker) [reply source/standards/dns-hosting.html.md.erb]: We are finishing off a migration away from Dyn towards AWS/Google but in principle, they are a valid supplier.

Comment (soupdragon99) [reply source/standards/dns-hosting.html.md.erb]: thanks for looking at that PR and your comment, let's leave the Dyn note in the guidance for now.

File: source/standards/dns-hosting.html.md.erb [hunk 3]
```diff
@@ -1,24 +1,23 @@
 ---
-title: DNS hosting
+title: How to manage DNS
 expires: 2018-01-01
```
Comment (Davidslv) [inline source/standards/dns-hosting.html.md.erb]: Would you be able to change this date to something in the future please?

**Comment by soupdragon99** [comment]
@Davidslv date extended by three months.

**Comment by soupdragon99** [comment]
Currently making changes to this guidance with input from @stephenharker

**Pull request merged**